Stride-Labs/stride
# V14 vesting migrations

**Summary**

ICS
- replace interchain security in go.mod (remove validation check)
- Send 40% of tokens in ConsToSendToProvider pool back to FeeCollector
- Add reward denoms (revert https://github.com/Stride-Labs/stride/compare/65a10d6dea3a95833c0df6d894a889219cd10d66...4b1c63332452b2772dc1b26b47547975b8cbd8e0) 

Vesting changes
- Integrate the module (pulled code from Evmos with no changes)
- Add store key
- Migrate account 1 to clawback account
- Migrate account 2 start date to Sept 4, 2022

**Test plan**

Added unittests for the following: 

ICS
- Check params after upgrade
- Check ConsToSendToProvider balance after upgrade

Vesting
- Check account type after upgrade
- Check vested tokens after upgrade
